### PR TITLE
Exabeam Data Connector

### PR DESCRIPTION
**Pull request by vu-socprime**
## Proposed Changes

  - Added Exabeam Data Connector

### CODE REVIEW

**Comment by preetikr** [comment]
@vu-socprime - Please fix for validation errors. Thanks.

**Comment by vu-socprime** [comment]
> @vu-socprime - Please fix for validation errors. Thanks.

@preetikr fixed

File: DataConnectors/Exabeam/Connector_Exabeam_Syslog.json [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+    ],
+    "sampleQueries": [
+        {
+            "description" : "Top 10 Clients (Source IP)",
+            "query": "ExabeamEvent\n | summarize count() by SrcIpAddr\n | top 10 by count_"
+        }
+    ],
+    "dataTypes": [
+        {
+            "name": "Syslog(Exabeam)",
```
Comment (v-ampami) [inline DataConnectors/Exabeam/Connector_Exabeam_Syslog.json:22]: Please give space between Syslog and Datatype.
eg: "Syslog (Exabeam)"

**Review by v-ampami — CHANGES_REQUESTED** [review]

File: DataConnectors/Exabeam/Connector_Exabeam_Syslog.json (hunk 1 above)
Comment (vu-socprime) [reply DataConnectors/Exabeam/Connector_Exabeam_Syslog.json]: done

File: DataConnectors/Exabeam/Connector_Exabeam_Syslog.json [hunk 2]
```diff
@@ -0,0 +1,156 @@
+{
+    "id": "Exabeam",
+    "title": "Exabeam Advanced Analytics",
+    "publisher": "Exabeam",
+    "descriptionMarkdown": "The [Exabeam Advanced Analytics](https://www.exabeam.com/ueba/advanced-analytics-and-mitre-detect-and-stop-threats/) data connector provides the capability to ingest Exabeam Advanced Analytics events into Azure Sentinel. Refer to [Exabeam Advanced Analytics documentation](https://docs.exabeam.com/) for more information.",
+    "additionalRequirementBanner": "This data connector depends on a parser based on Kusto Function to work as expected. Follow the steps to use this Kusto Function alias **ExabeamEvent** in queries and workbooks. [Follow steps to get this Kusto Function>](https://aka.ms/sentinel-Exabeam-parser)",
+    "graphQueries": [
+        {
+            "metricName": "Total data received",
+            "legend": "Syslog (Exabeam)",
```
Comment (v-ampami) [inline DataConnectors/Exabeam/Connector_Exabeam_Syslog.json:10]: Legend should be datatype_name which is Exabeam

File: DataConnectors/Exabeam/Connector_Exabeam_Syslog.json [hunk 3]
```diff
@@ -0,0 +1,156 @@
...
+    "sampleQueries": [
+        {
+            "description" : "Top 10 Clients (Source IP)",
+            "query": "ExabeamEvent\n | summarize count() by SrcIpAddr\n | top 10 by count_"
+        }
+    ],
+    "dataTypes": [
+        {
+            "name": "Syslog (Exabeam)",
+            "lastDataReceivedQuery": "ExabeamEvent\n            | summarize Time = max(TimeGenerated)\n            | where isnotempty(Time)"
```
Comment (v-jayakal) [inline DataConnectors/Exabeam/Connector_Exabeam_Syslog.json:23]: ExbeamEvent parser does not project (return) TimeGenerated, hence the query will fail. Can you please add TimeGenerated to the columns returned in the ExbeamEvent parser?

File: DataConnectors/Exabeam/Connector_Exabeam_Syslog.json [hunk 4]
```diff
@@ -0,0 +1,156 @@
...
+    "connectivityCriterias": [
+        {
+            "type": "IsConnectedQuery",
+            "value": [
+                "ExabeamEvent\n            | summarize LastLogReceived = max(TimeGenerated)\n            | project IsConnected = LastLogReceived > ago(30d)"
+            ]
+        }
+    ],
+    "availability": {
+        "status": 1,
```
Comment (v-jayakal) [inline DataConnectors/Exabeam/Connector_Exabeam_Syslog.json:35]: Status = 2, please change.

**Review by v-jayakal — CHANGES_REQUESTED** [review]

**Review by v-ampami — CHANGES_REQUESTED** [review]

File: DataConnectors/Exabeam/Connector_Exabeam_Syslog.json (hunk 3 above)
Comment (vu-socprime) [reply DataConnectors/Exabeam/Connector_Exabeam_Syslog.json:23]: updated

File: DataConnectors/Exabeam/Connector_Exabeam_Syslog.json (hunk 4 above)
Comment (vu-socprime) [reply DataConnectors/Exabeam/Connector_Exabeam_Syslog.json]: updated

File: DataConnectors/Exabeam/Connector_Exabeam_Syslog.json (hunk 2 above)
Comment (vu-socprime) [reply DataConnectors/Exabeam/Connector_Exabeam_Syslog.json]: updated

**Review by v-jayakal — APPROVED** [review]

**Pull request merged**